### ISSUE TITLE
Support for filecoin consensus in shards

### ISSUE THREAD

**Issue by adlrocha** [filecoin-project/eudico#40]
* Support native support of filecoin consensus for Eudico, i.e. we can `eudico filcns daemon` to start a eudico node running filecoin consensus.
* Support for Filecoin Expected Consensus in subnets.

We'll leave out of scope for now:
- Management of power in subnets.
- Adding sectors, mining and bookkeeping in hierarchical consensus.

**Comment by adlrocha** [comment]
A first attempt of this has been implemented here: https://github.com/filecoin-project/eudico/pull/57

This was done with the previous actor architecture of the hierarchical consensus and may require a re-design after #70 is merged. This new design may simplify things, and the aforementioned PR has the core foundation for this issue.

**Comment by adlrocha** [comment]
Doesn't apply anymore. We are not going to support Filecoin consensus in subnets.
